tests/withBlockTracker.ts

```diff
@@ -0,0 +1,252 @@
+import util from 'util';
+import SafeEventEmitter from '@metamask/safe-event-emitter';
+import { JsonRpcRequest, JsonRpcResponse } from 'json-rpc-engine';
+import { Provider } from '../src/BaseBlockTracker';
+import {
+  SubscribeBlockTracker,
+  SubscribeBlockTrackerOptions,
+} from '../src/SubscribeBlockTracker';
+import {
+  PollingBlockTracker,
+  PollingBlockTrackerOptions,
+} from '../src/PollingBlockTracker';
+
+interface WithPollingBlockTrackerTypes {
```

Comment:
Nit: This interface doesn't seem to be used. It's just grouping the two properties together. That seems like an unconventional way to use an interface. I do not find `WithPollingBlockTrackerTypes['withBlockTrackerCallback']` to be more readable than `WithPollingBlockTrackerCallback`

Reply:
Yeah, you're right. This was left over from trying to make a generic `withBlockTracker` function, but is not needed anymore. Change in [c3e20c9](https://github.com/MetaMask/eth-block-tracker/pull/103/commits/c3e20c989f39fad72cc1bc2f002839d6db4f61cd).